test/test_processor/test_config.py

```diff
@@ -161,7 +161,9 @@ def test_get_kwargs_constraints(self):
         }
 
         kwargs_constraints_2 = {
```

Comment:
Why is it necessary to add two more elements in `'joint_source_with_source'`?

Reply:
For the source light profile, I change the settings from  just 'shapelets' to 'sersic ellipse, shapelets'. This is added so that the gaussian source prior settings makes more sense

Reply:
Will there be a test error if you didn't add them. If you want the example to make more sense to users by being self-consistent, should not you do that in the config file, because that's what a user would look at? A user is never expected to look into the test functions, and you are changing the settings only in a test function.

Reply:
I did change the config file for lens_system3 to include  'sersic ellipse, shapelets'  too, that why I  also needed to change to test function.

Reply:
I have remove the extra elements from 'joint_source_with_source'. I use the deepcopy to add the 'prior_source' and 'prior_ps'

Reply:
Ok. But, what's the rationale behind injecting `'prior_source'` and `'prior_ps'` into the `config` object instead of having them in the settings file? You have the prior settings for lens and lens light in the config file, but not the others.

Reply:
Sorry for the confusion. I was hoping to make the lens_system3 config file to be fully working. Since lens_system3  only have 'shapelets' for its source light and have no point-source, I thought it doesnt make sense to add those priors into the settings file for  lens_system3

Reply:
That's ok, but you could still have `'prior_source'` specified in the settings file, no?

Reply:
You are right, I forgot about shapelets, I added the 'prior source' into the config file